mobile/src/components/authentication-guard.js

```diff
@@ -57,7 +58,7 @@ class AuthenticationGuard extends Component {
   }
 
   _hasAuthentication = () => {
-    return this.props.settings.biometryType || this.props.settings.pin
+    return this.props.settings.biometryType || this.props.settings.pinStatus
```

Comment:
Do we need to use the pinStatus setting, can we just use the truthy value of the pin? I believe this will break existing installations that have a pin set because the pinStatus value will be `null` even though they have a pin set.

Reply:
@tomlinton I have this design.
https://app.zeplin.io/project/59fa2311bac7acbc8d953da9/screen/5d92a607af15cb787cee6e77
if we use truthy value of pin and we disable pin authentication, we have to set truthy value with null. Then, we want to enable Pin authentication. We don't have value for Pin authentication. That is reason why i use pinStatus.
Do you have any ideas to change this ?

Reply:
Sorry, I saw micahalcorn 's idea. I will fix this issue.